mw_js_plugin.go

```diff
@@ -337,7 +337,12 @@ func (j *JSVM) Init(spec *APISpec) {
 	// Add environment API
 	j.LoadTykJSApi()
 
-	j.Timeout = 5 * time.Second
+	if config.Global.JSVMTimeout <= 0 {
+		j.Timeout = 5 * time.Second
```

Comment:
I'd add default timeout as a constant to avoid adding more "magic" numbers in code

Reply:
@dencoded Agree!